cve_bin_tool/OutputEngine.py

```diff
@@ -14,6 +14,12 @@ def __init__(self, filename=None, modules=None, logger=None):
         self.modules = modules
         self.logger = logger
 
+        # Max space for each console tabular cell
```

Comment:
These should probably be class local rather than instance local.


```python

class OutputEngine(object):
    MODULENAME_MAX = 18
    VERSION_MAX = 9
    CVE_NUMBER_MAX = 18
    CVE_SEVERITY_MAX = 9

    def __init__(self, filename=None, modules=None, logger=None):
```